<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="MySuite" duration-ms="65243" started-at="2015-04-19T18:46:07Z" finished-at="2015-04-19T18:47:12Z">
    <groups>
    </groups>
    <test name="Default Test" duration-ms="65243" started-at="2015-04-19T18:46:07Z" finished-at="2015-04-19T18:47:12Z">
      <class name="driver.TestDriver">
        <test-method status="FAIL" signature="test()" name="test" duration-ms="62875" started-at="2015-04-19T18:46:07Z" finished-at="2015-04-19T18:47:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[1. Check test case step for errors 2. Verify the behavior of the application manually]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 1. Check test case step for errors 2. Verify the behavior of the application manually
	at org.testng.Assert.fail(Assert.java:89)
	at driver.KeywordLauncher.executeIndexNull(KeywordLauncher.java:118)
	at driver.KeywordLauncher.execute(KeywordLauncher.java:80)
	at driver.KeywordLauncher.set(KeywordLauncher.java:70)
	at driver.KeywordLauncher.<init>(KeywordLauncher.java:34)
	at driver.TestDriver.test(TestDriver.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:842)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1166)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="test()" name="test" duration-ms="36015" started-at="2015-04-19T18:46:07Z" finished-at="2015-04-19T18:46:43Z">
        </test-method>
        <test-method status="PASS" signature="tearDown()" name="tearDown" is-config="true" duration-ms="1542" started-at="2015-04-19T18:46:43Z" finished-at="2015-04-19T18:46:45Z">
        </test-method>
        <test-method status="PASS" signature="tearDown()" name="tearDown" is-config="true" duration-ms="894" started-at="2015-04-19T18:47:12Z" finished-at="2015-04-19T18:47:12Z">
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
